packages/babel-plugin-dev/src/index.ts

```diff
@@ -0,0 +1,55 @@
+import  PathModule from 'path';
```

Comment:
import just as `path`

Reply:
I didn't want to get this confused with the 'path' variable that is used in the transform.  I suppose I could just `import { join } from 'path'` instead